src/contexts/parcelContext.js

```diff
@@ -26,19 +26,25 @@ const ParcelContextProvider = props => {
       return;
     }
     setSearching(true);
-    const response = await httpService.searchParcels(
-      parcelExtendedData.searchParams.statusFilter,
-      parcelExtendedData.searchParams.cityFilter,
-      parcelExtendedData.searchParams.nameFilter );
-    logger.log('[ParcelContextProvider] getAllparcelsfromDB response', response);
-    dispatch(loadParcels(response));
 
-    //need to query all parcels not only the search !
-    const cities = (await httpService.getParcelsCitiesDistinct()).sort();
-    dispatch(updateParcelsCities(cities));
-
-    setSearching(false);
-    setRefreshTime(refreshTime + 1);
+    try {
+      const response = await httpService.searchParcels(
+        parcelExtendedData.searchParams.statusFilter,
+        parcelExtendedData.searchParams.cityFilter,
+        parcelExtendedData.searchParams.nameFilter );
+      logger.log('[ParcelContextProvider] getAllparcelsfromDB response', response);
+      dispatch(loadParcels(response));
+  
+      //need to query all parcels not only the search !
+      const cities = (await httpService.getParcelsCitiesDistinct()).sort();
+      dispatch(updateParcelsCities(cities));
+  
+      setSearching(false);
+      setRefreshTime(refreshTime + 1);
+    } catch(e) {
+      logger.log(e);
+      dispatch(parcelsError(e));
+    }
   }
```

Comment:
I would like to suggest to add the new action to the list actions in the useEffect - even if does nothing (such as LOAD) just to make sure it did not slip out by mistake and also to document it.

Reply:
To which useEffect do you refer?

Reply:
useEffect of parcelContext

Reply:
Done.